www-root/content/js/brand.js

```diff
@@ -11,8 +11,7 @@ $(document).ready(function() {
 	    viewport = $(window),
 
 	    // get json
-		APIdc = '/api/?c=dc',
-	    APIstats = '/api/?c=gcs',
```

Comment:
Yeah gcs = get coin supply. It was more efficient to just calculate this in javascript so that is unused.